content/post/19-living-on-$1-day/index.md

```diff
@@ -0,0 +1,63 @@
+
+Another woman waited all day as well, and in her exit interview, when asked if she minded the wait, she smiled and said it hardly mattered compared to a doctor spending an hour with her.
+
+So what does all of this have to do with the title of this post, $1/day?  I actually think the residents of  Puerto Peñasco earn significantly more than that.  But when we got home tonight, I decided to watch the wonderfully done documentary [Living on One Dollar](http://livingonone.org/about/) on Netflix.
+
+It’s about four American college students who go to Guatemala for 60 days, give themselves an income equivalent to the local population (roughly a dollar per day), and see what it’s like to live in extreme poverty.
+
+In the movie, they live in the small town of Peña Blanca, where they experience such great generosity from people who have so little, and discover that small, incremental changes make all the difference at that level.
+
+There were two interesting parallels I saw between the residents of Puerto Peñasco and those of Peña Blanca.  First, in both cases, the natural temptation as a “wealthy” American is to pity the community, and to dwell on how many hardships they have.  But when you actually get into the community, you discover people who do not sit around feeling bad for themselves.  Instead, they have hopes, dreams, and desires like all of us, and they just want to get on with their lives.
+
+I like how my wife put it.  To pity them is to disrespect them.  Instead, if we’re there to help out, just focus on how we might be able to solve any problems they may be having.
+
+Second, I saw such intelligence and such a warm spirit in both communities.  Not one patient in the clinic complained, even those with serious maladies, sometimes multiple conditions at once.  And likewise in the village of Peña Blanca, the documentary displays a strong sense of community and a beautiful spirit of compassion among the locals.
+
+It made me think about the relative economics of it all.  The reality is that, in the case of Peña Blanca, I’ll earn more money in a single month than they will earn in 15 years.  It’s a nearly 200-to-1 earnings ratio.  Financially, there’s no contest.
+
+That feels unfair.  Out of that 200-to-1 ratio, how much of the 200 is due to any talents I might have versus simply the environment into which I was born, and the society in which I live, or even the “evolutionary paradigm” (more on that in a moment) in which I exist?  In other words, how much of that is just because of luck?
+
+I may understand the economic forces that give rise to that disparity, but how much did I do personally to be at the favorable end of them?
+
+And then there’s this somewhat complicated looking diagram.  You can click the image to see the full view, or just keep reading:
+
+![AQAL](https://s3.amazonaws.com/phxdevops-joshpadnick.com-images/wp-content/uploads/2015/06/spiral_dynamics_aqal_BIG-700x472.jpg)
```

Comment:
I don't want to refer to any third-party images. Can you download this image, store it in an `images/` folder in the same folder as this content (i.e. ` content/post/19-living-on-1-dollar-a-day/images/spiral_dynamics.jpg`), and then reference it using the `imgproc` shortcode:

```
{{< imgproc "images/violin.png" Fit "400x500" >}} [Image Source](http://pialogue.info/definitions/Spiral_Dynamics_Integral_SDi.php){{</imgproc>}}
```

Reply:
Sure